android/KMAPro/kMAPro/build.gradle

```diff
@@ -138,6 +157,25 @@ dependencies {
     }
 }
 
+// Use build-utils.sh to get common build version info.
+// Beware: Android Studio IDE Gradle sync will invalidate these values
+def executeBuildUtils() {
+    try {
+        exec {
+            commandLine 'bash', '-c', 'pwd; THIS_SCRIPT=build.gradle; . ../../../resources/build/build-utils.sh'
```

Comment:
> Is there any way to use the build-utils.sh environment status for -local, -test to determine local and PR builds? That would be really helpful!

I'm trying to go down that path but am running into some pain points:
1. Gradle calling a shell script will vary on WIndows vs Linux build agent (gosh!). atm this fails on Windows
2. The exported variables from build-utils.sh don't seem to persist when the Android Studio IDE does a Gradle sync (happens when installing the apk through the IDE).

Reply:
I also had tried putting this in version.gradle but that breaks the FV Android build (uses the exact same `android/version.gradle` file)

Reply:
Ah, since the parent shell has already called build-utils, hopefully all the vars are in the env.

The Gradle sync issue is still unresolved

Reply:
When there is a Gradle sync via IDE we can definitely just say 'local'.

Reply:
OK - I'll update so local / IDE build will say "Keyman Beta-local"
And for a PR build it would be "Keyman Beta-test-1234"